commons/src/components/MessageBody.svelte

```diff
@@ -58,8 +61,10 @@
 </style>
 
 <div>
-  {#if message && message.body}
-    {@html DOMPurify.sanitize(message.body)}
+  {#if message}
+    {#if typeof body === "string"}
```

Comment:
Instead of typeof string, can we check not null/undefined instead? 

```suggestion
    {#if body !== null}
```

This way if the body is a number or something (it won't be, but now it can be passed in!) it will still work. 
Additionally (my problem) I got really confused by the string check, whereas a null check wouldn't have made me think twice. 

If you like, please update the other instances of the string check as well :D